src/fmu/ensemble/realization.py

```diff
@@ -945,6 +969,13 @@ def get_eclsum(self, cache=True, include_restart=True):
             EclSum: object representing the summary file. None if
                 nothing was found.
         """
+        # pylint: disable=import-outside-toplevel
+        from .common import use_concurrent
+
+        if use_concurrent():
+            # In concurrent mode, caching is not used as
+            # we do not pickle the loaded EclSum objects
+            cache = False
```

Comment:
This change is significant. Each call to `ScratchRealization.get_eclsum()` is quite heavy, and many methods are written with several calls to `get_eclsum()`, such that the loss of caching might actually increase runtime even with concurrency.